portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,68 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Victor Zapata III's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Victor's Portfolio</h1>
+      <p>Hi my name is Victor Zapata III, and this is my portfolio. </p>
+      <p>About me:</p>
+      <p>I was born in New York City, where I have lived in my whole life. I really love living in a New York City thanks to the opportunities the city offers.</p>
+      <p>My history with my career started young when I was interested with architecture. Living in New York City where I can see many skyscaper, including the raising of the new World Trade Center builidng I was hook. Seeing the different and unique styles was fasinating to me. However, I saw this as a hobby not really as a career path I would love. So the other thing I really loved to do was math. Ever since Pre-K all the way up to now, math has been my favorite subject and not only did I succeed in it, but I loved doing it. The interesting ways to solve each problem had me hook. Having the choice how to solve a problem and getting the result you get was amazing. However, I notice there not a specific career I could pick that will allow me to do math in the way I love to do it.</p>
+      <p>It wasn't until High School I figure out the career path I know I will equally love and enjoy. When I took my AP Computer Science A course in my Junior year of High School. The course was focus around the coding language Java. Which prior to doing the class I played the game Minecraft. In the game I would make maps where it would require me to use in-game support to make these maps come to life. This eventually led me to modifying the game.</p> 
+      <p>Since the verison of Minecraft I was playing was Java based, a lot of commands, and when moding I would need to code using Java. So from my exposure with Minecraft I was able to understand the class and have the necessary problem solving skills that let me breeze thought the class. Which at the end of actual AP test I said I had fun, and I got a 5 on the exam.</p>
```

Comment:
```suggestion
      <p>Since the version of Minecraft I was playing was Java based, a lot of commands, and when modding I would need to code using Java. So from my exposure with Minecraft I was able to understand the class and have the necessary problem solving skills that let me breeze thought the class. Which at the end of actual AP test I said I had fun, and I got a 5 on the exam.</p>
```